<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormNewMap</class>
 <widget class="QWidget" name="FormNewMap">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>161</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>257</width>
    <height>161</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>257</width>
    <height>161</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New map</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lifetimeLabel">
     <property name="text">
      <string>Life time:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lifetimeEdit">
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="testCaseLabel">
     <property name="text">
      <string>Test case:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="testCaseEdit">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mapFilenameLabel">
     <property name="text">
      <string>File name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="mapFilenameEdit">
     <property name="text">
      <string>map/agent.map</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="selectMapFileButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>lifetimeEdit</tabstop>
  <tabstop>testCaseEdit</tabstop>
  <tabstop>mapFilenameEdit</tabstop>
  <tabstop>selectMapFileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
